tests/_core/test_models.py

```diff
@@ -53,7 +53,7 @@
 RNG = default_rng()
 
 # should be set to False for pre-commit and CI; useful to set to True for are fixing/debugging tests
-SHORT_TESTS = False
+SHORT_TESTS = True
```

Comment:
wonder if this is gonna produce inconsistencies across CI runs when it's left on. I think an option would be to add custom marks https://docs.pytest.org/en/7.1.x/example/markers.html

e.g. add `@pytest.mark.fast()` as decorator and then use `pytest -v -m fast`

Reply:
Well spotted! I have decorated some tests with `@pytest.mark.ci_only`, and now locally, when debugging, I run with `pytest -v -m "not ci_only"`